app/views/organisations/index.html.erb

```diff
@@ -11,7 +11,7 @@
 
   <tbody>
     <% @organisations.each do |organisation| %>
-      <tr>
+      <tr id='<%= organisation.slug %>-row'>
```

Comment:
slug is basically a human readable id, we are going to have another story to put in uniqueness validations.